tests/test_verifyStats.py

```diff
@@ -0,0 +1,264 @@
+            results = task.run(self.inputExp, self.camera)
+
+            # Or the detector stats
+            config.catalogStatKeywords = {}
+            config.detectorStatKeywords = {'DET_SIGMA', 'STDEV'}
+            task = cpVerify.CpVerifyStatsTask(config=config)
+            results = task.run(self.inputExp, self.camera)
+        self.assertIsNone(results)
+
+    def test_generic(self):
+        # Test a subset of the output values to identify that the
+        # image stat methods haven't changed.
+        config = cpVerify.CpVerifyStatsConfig()
+        config.imageStatKeywords = {'MEAN': 'MEAN', 'MEDIAN': 'MEDIAN', 'CLIPPED': 'MEANCLIP',
+                                    'SIGMA': 'STDEV'}
+        config.unmaskedImageStatKeywords = {'un_MEAN': 'MEAN', 'un_MEDIAN': 'MEDIAN',
+                                            'un_CLIPPED': 'MEANCLIP',
+                                            'un_SIGMA': 'STDEV'}
+        config.crImageStatKeywords = {'cr_MEAN': 'MEAN', 'cr_MEDIAN': 'MEDIAN', 'cr_CLIPPED': 'MEANCLIP',
+                                      'cr_SIGMA': 'STDEV'}
+        config.normImageStatKeywords = {'norm_MEAN': 'MEAN', 'norm_MEDIAN': 'MEDIAN',
+                                        'norm_CLIPPED': 'MEANCLIP',
+                                        'norm_SIGMA': 'STDEV'}
+        task = ToySubClass(config=config)
+
+        results = task.run(self.inputExp, self.camera)
+        resultStats = results.outputStats
+
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['MEAN'], 1506.06976, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['un_MEAN'], 1501.0299, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['norm_MEAN'], 301.213957, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['cr_MEAN'], 1504.2776, 4)
+
+        self.assertTrue(resultStats['VERIFY']['A REAL TEST'])
+        self.assertFalse(resultStats['VERIFY']['A BAD TEST'])
+
+        self.assertTrue(resultStats['SUCCESS'])
+
+
+class VerifyBiasTestCase(lsst.utils.tests.TestCase):
+    """Unit test for stats code.
+    """
+
+    def setUp(self):
+        # Generate a mock exposure/camera to test.
+        config = isrMock.IsrMockConfig()
+        config.isTrimmed = True
+        config.rngSeed = 12345
+        biasExposure = isrMock.BiasMock(config=config).run()
+
+        config.rngSeed = 54321
+        fakeBias = isrMock.BiasMock(config=config).run()
+
+        self.inputExp = biasExposure.clone()
+        mi = self.inputExp.getMaskedImage()
+        mi.scaledMinus(1.0, fakeBias.getMaskedImage())
+        makeMockExp(self.inputExp)
+
+        self.camera = isrMock.IsrMock().getCamera()
+
+    def test_bias(self):
+        # Test a subset of the output values to identify that the
+        # image stat methods haven't changed.
+        task = cpVerify.CpVerifyBiasTask()
+        results = task.run(self.inputExp, self.camera)
+        resultStats = results.outputStats
```

Comment:
Might be helpful to name these, e.g. `biasResults` & `biasStats`, so it's clear which set we're looking at in this function (and same for other calib-level tests below).